tutor/plugins/openedx.py

```diff
@@ -142,3 +142,71 @@ def is_directory_mounted(image_name: str, dirname: str) -> bool:
 hooks.Filters.ENV_TEMPLATE_VARIABLES.add_item(
     ("iter_mounted_directories", iter_mounted_directories)
 )
+
+
+@hooks.Filters.LMS_WORKER_COMMAND.add(priority=hooks.priorities.HIGH)
+def _add_default_lms_worker_parameters(
+    worker_configs: dict[str, str]
```

Comment:
Let's make this a `list[str]`, such that:

1. arguments are properly ordered
2. we can support value-less options, such as `--without-gossip`.

Also, `worker_configs` should be renamed to `command`.